### PR TITLE
feat: Use new cross chain transaction payload action

### PR DESCRIPTION
**Pull request by LautaroPetaccio**
This PR uses the new version of the transactions module, which fetches and waits for pending cross chain transactions.
It removes all the logic created to work with cross chain transactions once they were launched.

### CODE REVIEW

File: webapp/src/modules/item/sagas.ts [hunk 1]
```diff
@@ -296,67 +287,10 @@ export function* itemSaga(getIdentity: () => AuthIdentity | undefined) {
...
-          )
-        )
-      }
-    }
-  }
-
   function* handleBuyItemCrossChain(action: BuyItemCrossChainRequestAction) {
     const { item, route, order } = action.payload
     try {
+      console.log('Buying item cross chain')
```
Comment (juanmahidalgo) [inline webapp/src/modules/item/sagas.ts:293]: whoops

File: webapp/src/components/Modals/BuyWithCryptoModal/BuyWithCryptoModal.tsx [hunk 2]
```diff
@@ -378,7 +378,9 @@ export const BuyWithCryptoModal = (props: Props) => {
           data-testid={GET_MANA_BUTTON_TEST_ID}
           loading={isFetchingBalance || isBuyingAsset}
           onClick={() => {
-            onGetMana()
+            // onGetMana()
+            console.log("Getting MANA")
```
Comment (juanmahidalgo) [inline webapp/src/components/Modals/BuyWithCryptoModal/BuyWithCryptoModal.tsx:382]:
```suggestion
            onGetMana()
```

File: webapp/src/modules/item/sagas.ts (hunk 1 above)
Comment (LautaroPetaccio) [reply webapp/src/modules/item/sagas.ts]: Removed it!

File: webapp/src/config/env/dev.json [hunk 3]
```diff
@@ -44,6 +44,7 @@
   "SUBGRAPH_WORKER": "https://subgraph.decentraland.zone",
   "SENTRY_DSN": "https://1dc401149e1c819b8477565c9cdd9b70@o4504361728212992.ingest.sentry.io/4505743351676928",
   "SQUID_API_URL": "https://testnet.v2.api.squidrouter.com/",
+  "SQUID_RETRY_DELAY": "3000",
```
[the remark this answers is not in the document]
Comment (LautaroPetaccio) [reply webapp/src/config/env/dev.json]: Sure! Changed it to 2 secs.

**Pull request merged**